process-compose/suzuka-full-node/process-compose.yml

```diff
@@ -32,7 +32,7 @@ processes:
 
   suzuka-full-node:
     command: |
-      ./target/$CARGO_PROFILE/suzuka-full-node
+      RUST_LOG=debug ./target/$CARGO_PROFILE/suzuka-full-node
```

Comment:
Nitpick: it may be stylistically better to use the `env` configuration object in YAML.